tensorflow-framework/src/test/java/org/tensorflow/framework/data/MapDatasetTest.java

```diff
@@ -15,8 +15,9 @@
  */
 package org.tensorflow.framework.data;
 
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
Could you clean up the static imports so they are adjacent?

Reply:
Actually there were several cases of unused imports for JUnit. I used a sed script to make the JUnit changes, so these were there before, but I'll be glad to clean them up.